src/Microsoft.OpenApi/Models/OpenApiDiscriminator.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.OpenApi.Models
     /// <summary>
     /// Discriminator object.
     /// </summary>
-    public class OpenApiDiscriminator : IOpenApiSerializable
+    public class OpenApiDiscriminator : IOpenApiSerializable, IOpenApiExtensible
```

Comment:
Seems weird that this change is in this PR. But maybe this is 3.1 feature.

Reply:
This should have been a separate PR, that was an oversight on my end.